pkg/crc/validation/validation.go

```diff
@@ -58,18 +59,20 @@ func ValidateEnoughMemory(value int) error {
 }
 
 // ValidateBundlePath checks if the provided bundle path exist
-func ValidateBundlePath(bundle string) error {
-	if err := ValidatePath(bundle); err != nil {
+func ValidateBundlePath(bundlePath string) error {
+	if err := ValidatePath(bundlePath); err != nil {
 		if constants.BundleEmbedded() {
 			return fmt.Errorf("Run 'crc setup' to unpack the bundle to disk")
 		}
-		return fmt.Errorf("%s not found, please provide the path to a valid bundle using the -b option", bundle)
+		return fmt.Errorf("%s not found, please provide the path to a valid bundle using the -b option", bundlePath)
 	}
 
-	userProvidedBundle := filepath.Base(bundle)
-	if userProvidedBundle != constants.GetDefaultBundle() {
+	userProvidedBundle := filepath.Base(bundlePath)
+	customBundle := fmt.Sprintf("%s_custom", bundle.GetBundleNameWithoutExtension(constants.GetDefaultBundle()))
+	if userProvidedBundle != constants.GetDefaultBundle() ||
+		strings.Contains(userProvidedBundle, customBundle) {
```

Comment:
`strings.StartsWith`? But not sure about the conditional, I'd expect `!strings.Contains(...)` ?

Reply:
Forgot to mention, but would be good to only generate hte custom bundle name in a single place. Instead of `GetBundleNameWithoutExtension`, this could become `GetCustomBundleName()`. This avoids duplicating this generation in 2 very separate places (validation and generate_bundle)